dspace-api/src/main/java/org/dspace/content/service/DSpaceObjectService.java

```diff
@@ -370,6 +370,26 @@ public void setMetadataSingleValue(Context context, T dso, String schema, String
 
     public void delete(Context context, T dso) throws SQLException, AuthorizeException, IOException;
 
+    /**
+     * Add a single metadata field. Whether it's appended or prepended depends on index parameter.
+     * Use <code>clearDC</code> to remove values.
```

Comment:
I'm not seeing a method named `clearDC`?  Is this supposed to reference the `clearMetadata()` method?